src/bin/sol-fbp-runner/inspector.c

```diff
@@ -172,7 +172,9 @@ inspector_show_packet(const struct sol_flow_packet *packet)
             fprintf(stdout, "<\"%s\">", v);
             return;
         }
-    } else if (type == SOL_FLOW_PACKET_TYPE_BLOB) {
+    } else if (type == SOL_FLOW_PACKET_TYPE_BLOB ||
+        type == SOL_FLOW_PACKET_TYPE_JSON_OBJECT ||
+        type == SOL_FLOW_PACKET_TYPE_JSON_ARRAY) {
```

Comment:
meh, kinda bad to use the blob format for the inspector. You could do something like:

```c
if (type == SOL_FLOW_PACKET_TYPE_JSON_OBJECT || SOL_FLOW_PACKET_TYPE_JSON_ARRAY) {
    struct sol_blob *v;
    if (sol_flow_packet_get_blob(packet, &v) == 0) {
        fprintf(stdout, "%.*s", (int)v->size, (char *)v->data);
    }
}
```